examples/ota-requestor-app/ota-requestor-common/ota-requestor-app.matter

```diff
@@ -6,6 +6,168 @@ struct LabelStruct {
     char_string<16> value = 1;
 }
 
+server cluster Identify = 3 {
```

Comment:
This seems fine as far as it goes, but don't we need to actually compile in the relevant cluster impls?

Reply:
Yeah, these are automatically compiled in for the gn-based apps. I added it for esp32.